naucse/templates/_base.html

```diff
@@ -67,13 +67,7 @@
                 <div class="row">
                     <div class="col">
                         <h2>Zapoj se</h2>
-                        <p>
-                            {% if lesson is defined %}
-                                Uprav tuto stránku na <a href="https://github.com/pyvec/naucse.python.cz/blob/master/lessons/{{ page }}.{{ page.style }}">GitHubu</a>
-                            {% else %}
-                                Uprav tuto stránku na <a href="https://github.com/pyvec/naucse.python.cz">GitHubu</a>
-                            {% endif %}
-                        </p>
+                        <p>Uprav tuto stránku na <a href="{{ github_link }}">GitHubu</a></p>
```

Comment:
To be clear: here it's OK to use GitHub. It's the only place their human-readable name appears, one place to change should we need to switch, so no need parametrizing it.